README.md

```diff
@@ -1,330 +1,259 @@
-dataframe as an example. Otherwise hamilton can handle _any_ python object.
-
-Rather than thinking about manipulating a central object (dataframe in this case),
-you instead declare the components (columns in this case)/intermediate results you want to create, and the inputs that are required. There
-is no need for you to worry about maintaining this object, meaning you do not need to think about any "glue" code;
-this is all taken care of by the Hamilton framework.
-
-For example, rather than writing the following to manipulate a central dataframe object `df`:
-```python
-df['col_c'] = df['col_a'] + df['col_b']
-```
+Hamilton supports Python 3.8+. We include the optional `visualization` dependency to display our Hamilton DAG.
 
-you would write
-```python
-def col_c(col_a: pd.Series, col_b: pd.Series) -> pd.Series:
-    """Creating column c from summing column a and column b."""
-    return col_a + col_b
+```bash
+pip install "sf-hamilton[visualization]"
 ```
-In diagram form:
-![example](hamiltondag.png)
-The Hamilton framework will then be able to build a DAG from this function definition.
 
-So let's create a "Hello World" and start using Hamilton!
+To use the Hamilton UI, install the `ui` and `sdk` dependencies.
 
-### Your first hello world.
-By now, you should have installed Hamilton, so let's write some code.
+```bash
+pip install "sf-hamilton[ui,sdk]"
+```
 
-1. Create a file `my_functions.py` and add the following functions:
-```python
-import pandas as pd
+To try Hamilton in the browser, visit [www.tryhamilton.dev](https://www.tryhamilton.dev/?utm_source=README)
 
-def avg_3wk_spend(spend: pd.Series) -> pd.Series:
-    """Rolling 3 week average spend."""
-    return spend.rolling(3).mean()
+# Key Features
 
-def spend_per_signup(spend: pd.Series, signups: pd.Series) -> pd.Series:
-    """The cost per signup in relation to spend."""
-    return spend / signups
-```
-The astute observer will notice we have not defined `spend` or `signups` as functions. That is okay,
-this just means these need to be provided as input when we come to actually wanting to create a dataframe.
+Data teams write code to deliver business value, but few have the resources to standardize practices and provide quality assurance. Moving from proof-of-concept to production, and cross-function collaboration (e.g., data science, engineering, ops) remain challenging for teams big or small.
 
-Note: functions can take or create scalar values, in addition to any python object type.
+Hamilton bakes-in solutions to support you throughout a project's lifecycle:
 
-2. Create a `my_script.py` which is where code will live to tell Hamilton what to do:
+- **Built-in coding style**. Hamilton is built around Python functions encouraging  modular, easy to read, self-documenting, and unit testable code.
 
-```python
-import pandas as pd
-import my_functions
+- **Maintainable DAG definitions**.
```

Comment:
This has no definition? I'd take away "built-in-coding-style" and use that definition for "Maintainable DAG definitions". Seems overly opinionated at first glance.